ui/pages/swaps/create-new-swap/create-new-swap.js

```diff
@@ -23,21 +24,20 @@ export default function CreateNewSwap({ sensitiveTrackingProperties }) {
 
   return (
     <Box marginBottom={3} className="create-new-swap">
-      <a
-        href="#"
-        onClick={async (e) => {
-          e.preventDefault();
+      <button
+        onClick={async () => {
           trackEvent({
             event: 'Make Another Swap',
             category: EVENT.CATEGORIES.SWAPS,
             sensitiveProperties: sensitiveTrackingProperties,
           });
+          history.push(DEFAULT_ROUTE); // It cleans up Swaps state.
```

Comment:
Will this cause the component to dismount before invoking the two actions below?